job_runner/reporting/interactive.py

```diff
@@ -1,20 +1,27 @@
-from job_runner.reporting.message import Finish, Start
+import logging
+
+from job_runner.reporting.message import Message, Finish, Start
 from job_runner.reporting.base import Reporter
 
+logger = logging.getLogger(__name__)
+
 
 class Interactive(Reporter):
-    def report(self, msg):
+    def _report(self, msg: Message) -> str:  # type: ignore
         if isinstance(msg, Start):
-            print("Running job: {} ... ".format(msg.job.name()))
-        elif not msg.success() and not isinstance(msg, Finish):
-            print(
-                """failed ....
+            return f"Running job: {msg.job.name()} ... "
+        if isinstance(msg, Finish):
+            return (
+                "OK"
+                if msg.success()
+                else f"""failed ....
             -----------------------------------------------------------------
-            Error: {}
+            Error: {msg.error_message}
             -----------------------------------------------------------------
-            """.format(
-                    msg.error_message
-                )
+            """
             )
-        elif isinstance(msg, Finish) and msg.success():
-            print("OK")
+
+    def report(self, msg: Message):
+        msg_ = self._report(msg)
+        print(msg_)
+        logger.info(msg_)
```

Comment:
If the logging strategy should be this verbose, I think this line is a good approach. It logs what the reporter is doing without any interpretation. Maybe apply that to all logging for `job_runner`.

Reply:
Adjustment to the code was made base on the following thread.
https://github.com/equinor/ert/pull/2028#discussion_r712747404